src/Server.h

```diff
@@ -7,6 +7,7 @@
 #include <functional>
 #include <uv.h>
 #include <openssl/ossl_typ.h>
+#include <sys/un.h>
```

Comment:
This needs an `#ifdef` @alexhultman - maybe checking if it's not Windows is enough?